modules/currency-l0/src/main/scala/org/tessellation/currency/l0/cli/method.scala

```diff
@@ -82,6 +83,9 @@ object method {
 
     private val seedlistPathOpts: Opts[Option[Path]] = Opts.option[Path]("seedlist", "").orNone
 
+    val trustRatingsPathOpts: Opts[Option[Path]] =
+      Opts.option[Path]("ratings", "The path to the CSV of peer ID ratings").orNone
```

Comment:
I see this is duplicated everywhere. imo it would be great to define it once and reuse later on

Reply:
`genesisPathOpts`, `seedlistPathOpts`, and `trustRatingsPathOpts` have been centralized in `org.tessellation.sdk.cli.opts`.